src/drivers/launch/phidget_encoders.launch

```diff
@@ -0,0 +1,11 @@
+
+<launch>
+
+    <!-- Start the encoder node that gets encoder ticks over serial -->
+    <node name="encoders_driver" pkg="phidgets_high_speed_encoder" type="phidgets_high_speed_encoder" output="screen">
+        <rosparam param="joint0_name">left_encoder</rosparam>
```

Comment:
Document the numbering somewhere so everyone knows? Minor, but at some point we should have a way of referring to all sensors esp duplicated ones to match the indexing number we give them. Eg. From the back left to right side of robot is always 0 index start incrementing to the right. Minor, just thinking for confused future people.

Reply:
Good point. Added a quick comment to this launch file.